sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosEndToEndOperationLatencyPolicyConfigBuilder.java

```diff
@@ -21,6 +23,7 @@ public class CosmosEndToEndOperationLatencyPolicyConfigBuilder {
      */
     public CosmosEndToEndOperationLatencyPolicyConfigBuilder(Duration endToEndOperationTimeout) {
         this.endToEndOperationTimeout = endToEndOperationTimeout;
+        endToEndFeedOperationTimeout = endToEndOperationTimeout;
```

Comment:
since we are differentiate pointOperation vs nonPointOperations, so I think we should rename the `endToEndOperationTimeout` -> `endToEndPointOperationTimeout`, and then add a new setter method for `endToEndPointOperationTimeout`